tests/test_control_panel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from unittest import mock
+
+from tests import IntegrationTestCase
+from tests.fixtures import UserFactory
+
+
+@mock.patch('dashboard.service.fetch_user_data')
+class TestControlPanel(IntegrationTestCase):
+    def test_user_cannot_see_control_panel_in_navbar(self, fetch_data):
+        self.log_user_in(UserFactory.ROLE_USER, fetch_data)
+        response = self.client.get('/dashboard', follow_redirects=True)
+        self.assertNotIn(b'Control panel', response.data)
+
+    def test_admin_can_see_control_panel_in_navbar(self, fetch_data):
+        self.log_user_in(UserFactory.ROLE_ADMIN, fetch_data)
+        response = self.client.get('/dashboard', follow_redirects=True)
+        self.assertIn(b'Control panel', response.data)
+
+    def test_admin_can_see_control_panel_content(self, fetch_data):
+        self.log_user_in(UserFactory.ROLE_ADMIN, fetch_data)
+        response = self.client.get('/control', follow_redirects=True)
+        self.assertIn(b'master', response.data)
+        self.assertIn(b'START', response.data)
+        self.assertIn(b'STOP', response.data)
+
+    def test_user_cannot_see_control_panel_content(self, fetch_data):
+        self.log_user_in(UserFactory.ROLE_USER, fetch_data)
+        response = self.client.get('/control', follow_redirects=True)
+        self.assertNotIn(b'master', response.data)
+        self.assertNotIn(b'START', response.data)
+        self.assertNotIn(b'STOP', response.data)
+
+    def test_pipeline_not_present_in_config_is_not_displayed(self, fetch_data):
+        self.log_user_in(UserFactory.ROLE_ADMIN, fetch_data)
+        response = self.client.get('/control', follow_redirects=True)
+        self.assertNotIn(b'pipeline_name_not_in_config', response.data)
```

Comment:
I think this test has no sense. More sense will be when ``pipeline_name_not_in_config`` will be in zuul or ``status.json``, for now we can set here any random string and test. It's not make sense for me. 

For better, test if some pairs pipeline & branch from config are on page, in table.